src/gevent/monkey.py

```diff
@@ -119,6 +119,7 @@ def patch_psycopg(event):
 from __future__ import absolute_import
 from __future__ import print_function
 import sys
+from importlib import import_module
```

Comment:
I was concerned about this extra import either leading to more imports that might cause issue with monkey-patching, or with it taking time for cases that we might not need it. 

But on Python 3, it's imported by default anyway during startup, so those are completely moot.

On Python 2, it's a trivial file with no imports besides `sys`.